packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.tsx

```diff
@@ -40,64 +41,66 @@ export const TEXT_ALIGN = {
   start: 'start',
   center: 'center',
   end: 'end',
-};
+} as const;
 
 export type Props = {
-  className: ?string,
-  contentId: ?string,
-  tagline: ?string,
-  headline: string,
-  subheading: ?string,
-  sponsor: ?{
-    label: string,
-    logo: string,
-    altText: string,
-  },
-  buttonText: string,
-  onClick: () => void,
-  invertVertically: boolean,
-  textAlign: TEXT_ALIGN,
-  style: ?{},
+  className?: string | null;
+  contentId?: string | null;
+  tagline?: string | null;
+  headline: string;
+  subheading?: string | null;
```

Comment:
I would say, let's remove the `| null`, to avoid confusion. One `null` type is bad enough, but two is a bit too much. 😅

Reply:
Releasing in TypeScript will be a breaking change anyway, so I would opt for doing it right instead of Backwards compatibility. Also, we currently only use this in one place Banana within Skyscanner, so it won't need massive refactoring or anything.

Reply:
Let's do this in a second functional change PR, keeping this one a like-for-like change. 

Currently it is nullable as we default it to `null`, hence the type reflecting it.

Reply:
>so I would opt for doing it right instead of Backwards compatibility

I would disagree with this and we should right now ALWAYS be backwards compatible as if folks don't have chance to make changes or upgrades we risk them falling behind on Backpack versions because its seen as a burden